open-ce/build_env.py

```diff
@@ -73,8 +73,11 @@ def build_env(arg_strings=None):
     parser = make_parser()
     args = parser.parse_args(arg_strings)
 
+    print(utils.parse_arg_list(args.cuda_versions))
```

Comment:
This can be removed.

Reply:
ugh, had it removed, but didn't add the file to the last commit...it's in now